rs/ethereum/evm-rpc-client/src/types.rs

```diff
@@ -103,6 +103,45 @@ pub mod candid {
         pub topics: Option<Vec<Vec<String>>>,
     }
 
+    #[derive(Clone, Debug, PartialEq, Eq, CandidType, Deserialize)]
+    pub struct FeeHistoryArgs {
+        /// Number of blocks in the requested range.
+        /// Typically, providers request this to be between 1 and 1024.
+        #[serde(rename = "blockCount")]
+        pub block_count: u128,
+
+        /// Highest block of the requested range.
+        /// Integer block number, or "latest" for the last mined block or "pending", "earliest" for not yet mined transactions.
+        #[serde(rename = "newestBlock")]
+        pub newest_block: BlockTag,
+
+        /// A monotonically increasing list of percentile values between 0 and 100.
+        /// For each block in the requested range, the transactions will be sorted in ascending order
+        /// by effective tip per gas and the corresponding effective tip for the percentile
+        /// will be determined, accounting for gas consumed.
```

Comment:
Based on this comment, it is not entirely clear to me what `reward_percentiles` contains. Since it is "percentile values between 0 and 100", is the length of the vector always exactly 101?

Why are the transactions "sorted in ascending order by effective _tip per gas_"? The order might change quite a bit when computing the actual effective tips. The percentiles are derived from the effective tips and not the tips per gas, right?